.github/templates/linux_ci_workflow.yml.j2

```diff
@@ -564,6 +606,19 @@ jobs:
           name: pytorch_github_io
           if-no-files-found: error
           path: pytorch_github_io.zip
+      - name: Hold runner for 2 hours or until ssh sessions have drained
+        # Always hold for active ssh sessions
+        if: always()
+        run: |
+          echo "Holding runner for 2 hours until all ssh sessions have logged out"
+          for _ in $(seq 1440); do
+            # Break if no ssh session exists anymore
+            if [[ "$(who)" = "" ]]; then
+              break
+            fi
+            echo "."
+            sleep 5
+          done
```

Comment:
looks like this same `run` script is used in a few different places; would it make sense to put it into `.github/scripts` and just call it from here?